testing/main_test.go

```diff
@@ -90,6 +90,18 @@ func initORMApp(app application, dbURL *url.URL) (func() error, error) {
 		if err := cmd.Wait(); err.Error() != "signal: "+syscall.SIGKILL.String() {
 			return err
 		}
+
+		// Killing a process is not instant. For example, with the Hibernate server,
```

Comment:
As discussed, this comment doesn't tell the whole story. If you could make it a bit more information, that'd be good, but don't consider this a blocker.